Nodejs/Product/Nodejs/Microsoft.NodejsTools.targets

```diff
@@ -57,7 +57,7 @@
   <Target Name="CopyFilesToOutputDirectory"/>
 
   <Target Name="CoreCompile">
-    <Error Condition="!Exists('$(VSToolsPath)\TypeScript\Microsoft.TypeScript.targets') And '$(EnableTypeScript)' == 'true'" Text="TypeScript is not installed but TypeScript support is enabled in this project.  Please download and install TypeScript http://go.microsoft.com/fwlink/?LinkID=393159&amp;clcid=0x409."></Error>
+    <Error Condition="!Exists('$(VSToolsPath)\TypeScript\Microsoft.TypeScript.targets') And '$(EnableTypeScript)' == 'true' And '$(VisualStudioVersion)' != '15.0'" Text="TypeScript is not installed but TypeScript support is enabled in this project.  Please download and install TypeScript http://go.microsoft.com/fwlink/?LinkID=393159&amp;clcid=0x409."></Error>
```

Comment:
Ahh, I see. "Import Project" includes the fallback msbuild behavior, but not errors/messages because they are less constrained in their use case. Given that restriction, this appears to be the correct fix, though we should probably add a comment to clarify.